src/components/Products/ProductsToolbar/ProductsToolbar.module.scss

```diff
@@ -0,0 +1,105 @@
+@import '@style/_media';
+@import '@style/_mixins';
+
+.container {
+  padding: 16px 0px;
+
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  gap: 10px;
+
+  background-color: var(--clr-primary);
+}
+
+.sort {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+
+  & > div {
+    flex-direction: row;
+    align-items: center;
+    gap: 24px;
+
+    & > div {
+      padding: 0;
+      border: none;
+    }
+
+    & > div:first-child {
```

Comment:
In my opinion, it's better to avoid such selectors with divs

Reply:
Removed nested selectors